docs/architecture/adr-017-subspace-fee-grant.md

```diff
@@ -40,61 +41,87 @@ Currently, the `x/auth` module provides a `DeductFeeDecorator` that deducts the
 
 The `x/feegrant` module already provides an interface (`FeeAllowanceI`) to represent a generic allowance, along with a set of useful implementations like `BasicAllowance`, `AllowedMsgAllowance` and `PeriodicAllowance`. Since we are not going to introduce new kinds of allowances, nor we are going to edit how an allowance is represented, we will reuse these types.
 
-#### User Grant
+#### Grant
 
-The `x/feegrant` module provides a `Grant` object used to store the `granter`, `grantee` and what kind of `allowance` is granted to a specific user. Since this is enough information for us as well, we will use the same object to store user-related fee grants.
+In order to represent a fee grant granted to a user or user group, we will implement a custom `Grant` type. This will contain the `granter`, `grantee`, which is the interface to include user and user group, and what kind of `allowance` is granted to the grantee:
```

Comment:
```suggestion
In order to represent a fee grant granted to a user or user group, we will implement a custom `Grant` type. This will contain a `granter` field, a `grantee` (an interface to represent either a user or user group), and what kind of `allowance` is granted to the grantee:
```